janusgraph-dist/src/test/java/org/janusgraph/pkgtest/TestCategory.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.pkgtest;
+
+public class TestCategory {
```

Comment:
There is also a TestCategory in janusgraph-backend-testutils module.

Reply:
It is container for constants. I could rename it to PkgTestCategory